ibazel/command/notify_command_test.go

```diff
@@ -69,11 +69,6 @@ func TestNotifyCommand(t *testing.T) {
 		[]string{"Build", "//path/to:target"},
 	})
 
-	err = c.stdin.Close()
```

Comment:
Why are you no longer closing the stdin pipe after writing to it? This diff is what's causing the test to time out and fail; the notify command is blocking on the read.

Reply:
Fixed